.env

```diff
@@ -0,0 +1 @@
+RWD__DOCKER_ZICHT_NL__PHP_COMPOSER=7.2-1.8.5
```

Comment:
Dit hoeft niet in een bundle

Reply:
Als je in de bundel `composer install && composer run-script test` wilt draaien dan heb je dit nodig

Reply:
Er zijn ook wel eens docker-compose.yml files in (PRs naar) bundles gekomen. Deze hebben we er ook allemaal uit gehaald.

Reply:
> Er zijn ook wel eens docker-compose.yml files in (PRs naar) bundles gekomen. Deze hebben we er ook allemaal uit gehaald.

Dan was dat ook niet zo handig dan... er zijn toch echt diverse bundles die ik los van een project op mijn disk heb staan waar ik ook linters en tests op los wil laten... dan is dit bestandje nodig in onze docker setup...